mesh_sequence_controller.py

```diff
@@ -37,6 +37,25 @@
 import glob
 from bpy.app.handlers import persistent
 
+import re
+
+def tryint(s):
+    try:
+        return int(s)
+    except:
+        return s
+
+def alphanum_key(s):
+    """ Turn a string into a list of string and number chunks.
+        "z23a" -> ["z", 23, "a"]
+    """
+    return [ tryint(c) for c in re.split('([0-9]+)', s) ]
+
+def sort_nicely(l):

```

Comment:
Let's change this to `sortNatural` and change `l` to something more descriptive

Reply:
This function was also removed.